config/vufind/markdown.ini

```diff
@@ -35,8 +35,11 @@
 ;     Attributes, ExternalLink, Footnote, HeadingPermalink, Mention, SmartPunct,
 ;     TableOfContents
 ; More about extensions: https://commonmark.thephpleague.com/1.5/extensions/overview/
-; Some of them could have a configuration, see sections bellow
-;extensions = Attributes,ExternalLink,Footnote,HeadingPermalink,Mention,SmartPunct,TableOfContents
+; Some of them could have a configuration, see sections bellow. Available extensions
+; are: Attributes, Autolink, DisallowedRawHtml, ExternalLink, Footnote,
+; HeadingPermalink, Mention, SmartPunct, Strikethrough, Table, TableOfContents,
+; TaskList.
+extensions = Autolink,DisallowedRawHtml,Strikethrough,Table,TaskList
```

Comment:
Now that we have a default list of extensions, do user who have customized this setting previously need to make any changes for things to work correctly? I doubt that this is being heavily used, but we should at least put a note in the changelog if some action may be necessary.

Reply:
Also, if the setting is commented out, should we now default to this default set, or should we continue to default to an empty array in the factory code?

Reply:
A note in changelog would be fine. To continue thing work as before, this set of extensions should be enabled.

Reply:
I made these extensions a default set if no extensions are set. To remove the default, and use raw commonmark, there is posibility to set the 'extensions' option to empty string

Reply:
Note added -- thanks!

Reply:
(And I revised my note to indicate that manual action is only needed for users who already customized the extensions setting in markdown.ini -- probably a very small number, if not zero). :-)